codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeSpecUtils.java

```diff
@@ -70,10 +74,23 @@ public ClassName providerInterfaceName() {
         return ClassName.get(basePackage(), intermediateModel.getMetadata().getServiceName() + "AuthSchemeProvider");
     }
 
+    public ClassName endpointDelegateName() {
```

Comment:
I think we should use EndpointRulesSpecUtils directly instead.

Reply:
We already do, I will remove this one that's not used.